modules/log6x-cluster-logging-collector-log-forward-syslog.adoc

```diff
@@ -0,0 +1,122 @@
+// Module included in the following assemblies:
+//
+// * observability/logging/logging-6.2/log6x-clf-6.2.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="cluster-logging-collector-log-forward-syslog-6x_{context}"]
+= Forwarding logs using the syslog protocol
+
+You can use the syslog link:https://tools.ietf.org/html/rfc3164[RFC3164] or link:https://tools.ietf.org/html/rfc5424[RFC5424] protocol to send a copy of your logs to an external log aggregator that is configured to accept the protocol instead of, or in addition to, the default Elasticsearch log store. You are responsible for configuring the external log aggregator, such as a syslog server, to receive the logs from {product-title}.
+
+To configure log forwarding using the syslog protocol, you must create a `ClusterLogForwarder` custom resource (CR) with one or more outputs to the syslog servers, and pipelines that use those outputs. The syslog output can use a UDP, TCP, or TLS connection.
+
+.Prerequisites
+
+* You must have a logging server that is configured to receive the logging data using the specified protocol or format.
+
+.Procedure
+
+. Create or edit a YAML file that defines the `ClusterLogForwarder` CR object:
++
+[source,yaml]
+----
+apiVersion: observability.openshift.io/v1
+kind: ClusterLogForwarder
+metadata:
+  name: collector
+spec:
+  managementState: Managed
+  outputs:
+  - name: rsyslog-east # <1>
+    syslog:
+      appName: <app_name> # <2>
+      enrichment: KubernetesMinimal
+      facility: <facility_value> # <3>
+      msgId: <message_ID> # <4>
+      payloadKey: <record_field> # <5>
+      procId: <process_ID> # <6>
+      rfc: <RFC3164_or_RFC5424> # <7>
+      severity: informational # <8>
+      tuning:
+        deliveryMode: <AtLeastOnce_or_AtMostOnce> # <9>
+      url: <url> # <10>
+    tls: # <11>
+      ca:
+        key: ca-bundle.crt
+        secretName: syslog-secret
+    type: syslog
+  pipelines:
+  - inputRefs: # <12>
+    - application
+    name: syslog-east # <13>
+    outputRefs:
+    - rsyslog-east
+  serviceAccount: # <14>
+    name: logcollector
+----
+<1> Specify a name for the output.
+<2> Optional: Specify the value for the "APP-NAME" part of the syslog message header. The value must conform with link:https://datatracker.ietf.org/doc/html/rfc5424[The Syslog Protocol]. The value can be a combination of static and dynamic values consisting of field paths followed by `||`, and then followed by another field path or a static value. The maximum length of the final values is truncated to 48 characters. You must encase a dynamic value curly brackets and the value must be followed with a static fallback value separated with `||`. Static values can only contain alphanumeric characters along with dashes, underscores, dots and forward slashes. Example value: <value1>-{.<value2>||"none"}.
```

Comment:
```suggestion
<2> Optional: Specify the value for the `appName` part of the syslog message header. The value must conform with link:https://datatracker.ietf.org/doc/html/rfc5424[The Syslog Protocol]. The value can be a combination of static and dynamic values consisting of field paths followed by `||`, and then followed by another field path or a static value. The maximum length of the final values is truncated to 48 characters. You must encase a dynamic value curly brackets and the value must be followed with a static fallback value separated with `||`. Static values can only contain alphanumeric characters along with dashes, underscores, dots and forward slashes. Example value: <value1>-{.<value2>||"none"}.
```

Reply:
Whenever you reference inline parameters, you should use backticks: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference